pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+
+[tool.ruff]
+exclude = ["nucypher_ops/__init__.py"] # false positives
+select = ["E", "F", "I"]
+line-length = 180
```

Comment:
Pretty long line length 😅 - although it's hard to specify a value that everyone agrees on hehe.

Reply:
I prefer shorter lines myself, but it would be very much out of style with the current code style in `nucypher-ops`

Reply:
👍. Another solution is to ignore line length errors (https://docs.astral.sh/ruff/rules/line-too-long/) i.e. add this to the [tool.ruff] section:
```
ignore = ["E501"]
```

then you can remove the `line-length` setting. Up to you though.